Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -173,6 +194,59 @@ Object.defineProperties(ModelExperimentalSceneGraph.prototype, {
 function initialize(sceneGraph) {
   const components = sceneGraph._modelComponents;
   const scene = components.scene;
+
+  computeModelMatrix(sceneGraph);
+
+  const nodes = components.nodes;
+  const nodesLength = nodes.length;
+  sceneGraph._runtimeNodes = new Array(nodesLength);
```

Comment:
If I'm reading this code correctly, this array has room for all the nodes in the glTF, but the way the scene graph is traversed, only the ones used in the main `scene` are populated?

Probably okay, but it would be good to put a comment explaining this.

Reply:
Yes -- I can see how this would be unideal for glTFs with multiple scenes, but I'm not sure how to refactor this in a way that doesn't involve a map of indices the glTF to indices in the runtime array (unless that is preferred?)